pkg/internal/webconfig.yaml

```diff
@@ -0,0 +1,3 @@
+tls_server_config:
```

Comment:
The name of this file makes it unclear of its purpose.
Please name it `windows-exporter-config.yaml` or something similar.